Resources/Prototypes/Catalog/Fills/Lockers/service.yml

```diff
@@ -91,20 +91,22 @@
   components:
   - type: StorageFill
     contents:
-      - id: ClothingHandsGlovesLeather
-      - id: ClothingOuterApronBotanist
-      - id: ClothingHeadBandBotany
-      - id: HydroponicsToolClippers
-      - id: HydroponicsToolSpade
-      - id: HydroponicsToolMiniHoe
+#      - id: ClothingHandsGlovesLeather # Frontier
+#      - id: ClothingOuterApronBotanist # Frontier
+#      - id: ClothingHeadBandBotany # Frontier
+#      - id: HydroponicsToolClippers # Frontier
+#      - id: HydroponicsToolSpade # Frontier
+#      - id: HydroponicsToolMiniHoe # Frontier
```

Comment:
botanists still need tools, dont remove these

Reply:
All tools in vends / boxes.
I also added the missing ones.

Reply:
i think that the essential tools should be in lockers. The clothes can all get 1984'd out and the fluff, but the essential job tools should be like the only reason lockers exist.

The mindset behind that is, if a monkey wants to do the job, they cant actually buy the essential tools, they would need to ask a human to buy them, so they do need just the essentials for the locker.

Reply:
Added to a filled belt, filled belt added to locker.